.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,8 +7,9 @@
 
 Please ensure you've completed all of the following.
 
-- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
-- [ ] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
+- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
+- [ ] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
+- [ ] I update the version package [Updating](https://github.com/ferdium/ferdium-recipes/blob/main/docs/updating.md#4-updating-the-version-number) 
```

Comment:
Ah this is cool, then more ppl will hopefully do the version update right away without having to get reminded.

Reply:
Yes! I would've appreciated knowing about this prior to the commit hook yelling at me about it